### PR TITLE
'Consensus and MetaMask updates' feature

### PR DESCRIPTION
**Pull request by nickeskov**

### CODE REVIEW

File: pkg/settings/feature_string.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// Code generated by "stringer -type=Feature"; DO NOT EDIT.
```
Comment (alexeykiselev) [inline pkg/settings/feature_string.go:1]: Why do we need `String`? No usages, internal name not equal to description.

File: pkg/settings/features.go [hunk 2]
```diff
@@ -16,13 +19,18 @@ const (
...
-	InvokeExpression // 18
+	BlockReward           // 14
+	BlockV5               // 15
+	RideV5                // 16
+	RideV6                // 17
+	ConsensusImprovements // 18
+	InvokeExpression      // 19
 )
 
+func (f Feature) Description() string {
```
Comment (alexeykiselev) [inline pkg/settings/features.go:30]: Why do we need `Description`. No usages again.

Comment (alexeykiselev) [reply pkg/settings/features.go:30]: `Implemented` would be useful. But description only used in API in iteration over all features.
I think such functions should be introduced along with code that use them.

**Review by alexeykiselev — CHANGES_REQUESTED** [review]

File: pkg/settings/feature_string.go (hunk 1 above)
Comment (nickeskov) [reply pkg/settings/feature_string.go]: Fixed

File: pkg/settings/features.go (hunk 2 above)
Comment (nickeskov) [reply pkg/settings/features.go]: Agree, fixed

**Review by alexeykiselev — APPROVED** [review]

**Pull request merged**